api/ratelimit.proto

```diff
@@ -0,0 +1,102 @@
+syntax = "proto3";
+
+option go_package = "ratelimit";
+
+option cc_generic_services = true;
+
+package envoy.api.v1;
```

Comment:
forgot to tell you in person. But yeah this plan sounds good. Then we can migrate envoy and ratelimit, when I have gone through the process of making sure our internal clients can take the name change.